content/pieces/uncivilizing-digital-territories/index.md

```diff
@@ -183,3 +183,5 @@ But the fact is that most communities have already been exposed to the corporate
 All that is much easier said than done, as encouraging alternatives to mainstream usage of technology is, many times, like trying to swim against a strong current. But we can build networks of communities where experiences are shared, and principles of autonomy—and livelihoods which operate in mutual relationship with Mother Earth’s complex, abundant world—can be valued and celebrated. Building these relationships with native-cultures without being extractive is fundamental for learning how to reconnect with our essence and understanding more about non-civilized practices and ways of life.
 
 We might not be able to change the downhill course of civilization. But we can assist in strengthening our communities so that, when crises occur, we respond with our networks from a place of autonomy, solidarity, and resilience.
+
+_Editors' Note: This piece has been edited to remove a mention and citation of a figure who has aligned themselves with an organization that has expressed violent, transphobic beliefs. Therefore we have retracted the reference as to not signal support of this group._
```

Comment:
_Editors' Note: This piece has been edited to remove mention of a figure who has aligned themselves with violent, transphobic beliefs. We retract any reference that inadvertently signaled support of him, or affiliated groups._